apps/sms/services/js/bridge_service_mixin.js

```diff
@@ -55,6 +55,10 @@
         });
       }
 
+      if (this.onConnected) {
```

Comment:
It's simple and clear like other mixin method(initService), so let's keep it as it is.